README.md

```diff
@@ -1,56 +1,45 @@
 
-We’d like you to fetch this data asynchronously from our [Content API](https://open-platform.theguardian.com/explore/). Please do this on the client-side, your solution should run in the browser alone with no server-side requirements.
+You should fetch this data asynchronously from our [Content API](https://open-platform.theguardian.com/explore/). **Please do this on the client-side.** Your solution should run in the browser alone with no server-side requirements.
 
-At the Guardian we try to build things to be lightweight, accessible, and usable across browsers, devices and network conditions. When working on your application you should therefore focus on:
+### Considerations
 
-- Keeping the application as lightweight as possible.
-- Making it work across a range of device sizes.
-- Using semantically correct HTML.
+At the Guardian we try to build things to be lightweight, accessible, and usable across browsers, devices and network conditions. 
 
-### Optional requirements
+During implemetation, please choose 2 or 3 of the below considerations to focus on. Let us know which ones you have chosen in your repo's README.
 
 #### Accessibility
 
-A great deal of web content can be made accessible just by making sure the correct HTML elements are used for the correct purpose at all times. We'd like you consider how the HTML can be used to ensure maximum accessibility for our user's that rely on screen reading technology and/or keyboard navigation to access our content.
+A great deal of web content can be made accessible simply by using semantically correct HTML elements and role attributes. We'd like you to ensure the component is accessible for our users that rely on screen reading technology or keyboard navigation to access our content.
 
 #### Browser support
 
-Our site uses modern web standards to give the best possible reading experience for our visitors. Although we aim to be readable on all possible devices and browsers, not all are capable of running the full set of features. We'd like you to make sure your application works on older browsers as well as the latest browsers. Our [reccommended browser list](https://www.theguardian.com/help/recommended-browsers) states the browsers we currently support.
+Our site uses modern web standards to provide the best possible reading experience for our visitors. Although we aim to be readable on all possible devices and browsers, not all are capable of running the full set of features. We'd like you to make sure your application works on older browsers as well as the latest browsers. Our [recommended browser list](https://www.theguardian.com/help/recommended-browsers) states the browsers we currently support.
 
 #### No JavaScript
```

Comment:
No sure what to do about this consideration. I don't see how we can offer "a meaningful experience when JavaScript is unavailable" without a server side requirement

Reply:
Agreed, it seems bit much expecting someone to implement a server side solution in the event of no-js within the time limit. The requirement could be alluding to using some kind of CSS fallback for tabs _if_ the tabs list had been server side rendered and no-js was running in the browser? But it's difficult to describe that without it being confusing.